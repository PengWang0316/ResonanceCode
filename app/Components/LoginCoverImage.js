import React from 'react';

import * as coverImage from '../imgs/cover.png';

const LoginCoverImage = _ => (
  <div className="container text-center"><img className="img-fluid" src={coverImage.default} alt="Cover Background" /></div>
);
export default LoginCoverImage;
