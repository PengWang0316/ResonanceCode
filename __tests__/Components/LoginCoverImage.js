import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

import LoginCoverImage from '../../app/Components/LoginCoverImage';

describe('LoginCoverImage test', () => {
  // const defaultProps = {};
  // const getShallowComponent  = (props = defaultProps) => shallow(< {...props} />);

  test(' snapshot', () => expect(renderer.create(<LoginCoverImage />).toJSON()).toMatchSnapshot());
});
