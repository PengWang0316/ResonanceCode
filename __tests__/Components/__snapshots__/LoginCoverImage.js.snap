// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginCoverImage test  snapshot 1`] = `
<div
  className="container text-center"
>
  <img
    alt="Cover Background"
    className="img-fluid"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</div>
`;
